torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -311,6 +311,20 @@ RegisterOperators reg(
            pack(stack, t.sizes().vec());
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::EnumName(AnyEnumType enum) -> str",
+         [](Stack* stack) {
+           IValue e = pop(stack);
+           push(stack, e.toEnumHolder()->name());
+         },
+         aliasAnalysisFromSchema()),
+     Operator(
+         "prim::EnumValue(AnyEnumType enum) -> Any",
```

Comment:
The types of the IR values corresponding to `enum` and the return value are more narrow than `AnyEnumType` and `Any`. I think there is nothing we can do about `AnyEnumType` because those types are user-defined, but is there a restriction on underlying storage type? Can we use that to define more specific operators? E.g.
`prim::EnumValue(AnyEnumType enum) -> int`
`prim::EnumValue(AnyEnumType enum) -> float`
`prim::EnumValue(AnyEnumType enum) -> bool` (lol)